esp/src/src-react/components/Files.tsx

```diff
@@ -110,6 +112,23 @@ export const Files: React.FunctionComponent<FilesProps> = ({
     const [viewByScope, setViewByScope] = React.useState(false);
     const [uiState, setUIState] = React.useState({ ...defaultUIState });
 
+    const toggleTimezone = () => {
+        setUIState((prevState) => ({ ...prevState, isUTC: !prevState.isUTC }));
+      };
+    
+      useEffect(() => {
+        localStorage.setItem("isUTC", JSON.stringify(uiState.isUTC));
+      }, [uiState.isUTC]);
+    
+      useEffect(() => {
+        const storedIsUTC = JSON.parse(localStorage.getItem("isUTC"));
+        if (storedIsUTC !== null) {
+          setUIState((prevState) => ({ ...prevState, isUTC: storedIsUTC }));
+        }
+      }, []);
+    
+      const currentTime = uiState.isUTC ? Utility.convertToUTCTime(WsDfu.DFUInfo.Modified) : Utility.convertToLocalTime(WsDfu.DFUInfo.Modified);
```

Comment:
This should be something like:

<pre>
const currentTime = React.useCallback(timestamp => {
    return uiState.isUTC ? Utility(convertToUTCTime(timestamp) : Utility.convertToLocalTime(timestamp);
}, [uiState.isUTC]);
</pre>